azure-pipelines.yml

```diff
@@ -1,69 +0,0 @@
-schedules:
-- cron: "0 22 * * *"
-  displayName: Master nightly build
-  branches:
-    include:
-    - master
-  always: true
```

Comment:
This functionality is lost now. We should keep it as it is a nice mechanism to detect when for example a new Python breaks on Arch.

Reply:
Ah, i'll see if I can get it re-introduced. I assume one can trigger on both push and timer.

Reply:
Fixed now.